pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -372,6 +372,7 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 		{Name: "CEPH_ARGS", Value: "-m $(ROOK_CEPH_MON_HOST)"},
 		blockPathEnvVariable(osd.BlockPath),
 		cvModeEnvVariable(osd.CVMode),
+		dataDeviceClassEnvVar(osd.DeviceClass),
```

Comment:
Does ceph use this env var to assign the device class? More commonly we have used the command args for passing config to ceph. See the `args` slice below.

Reply:
When updating existing deployment OsdInfo getting created from ENV vars (from template container[0] vars) as i see. And that's why i added that var.

Reply:
Anyway it can be helpful info on the pod spec for troubleshooting, but I'm not sure it's strictly needed

Reply:
Check this https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/osd/osd.go#L452 - osdInfo is creating from Deployment info, including env vars . I'm not sure that we may put DeviceClass info in different place then env vars.